aspnetcore/fundamentals/index.md

```diff
@@ -264,6 +264,18 @@ For more information, see [Content root](xref:fundamentals/host/web-host#content
 
 The web root (also known as *webroot*) is the base path to public, static resources, such as CSS, JavaScript, and image files. The static files middleware will only serve files from the web root directory (and sub-directories) by default. The web root path defaults to *{Content Root}/wwwroot*, but a different location can be specified when [building the host](#host).
 
+::: moniker range=">= aspnetcore-3.0"
+
+For more information, see [ContentRootPath](/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.0#contentrootpath)
```

Comment:
content root is a different thing than web root -- the 3.0 link should go to web root as well, just in the generic-host doc.

Reply:
Just saw your message show up as I merged and the screen redrew!  fixing!